src/index.html

```diff
@@ -5,9 +5,37 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card">
+      <div class="card__photo"></div>
+
+      <p class="card__title">APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</p>
+
+      <p class="card__code">Product code: 195434</p>
+
+      <div class="card__review">
+        <div class="stars stars--5">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <p class="card__number-of-reviews">Reviews: 5</p>
+      </div>
+
+      <div class="card__price">
+        <p class="card__price-text">Price:</p>
+        <p class="card__price-number">$2,199</p>
+      </div>
+
+      <a href="#" class="card__buy">Buy</a>
```

Comment:
Read the task carefully:
- add `data-qa="hover"` attribute to the link (button)